src/reports/models/database/cassandra/cassandraConnector.js

```diff
@@ -8,6 +8,7 @@ let client;
 const isRowAppliedField = '[applied]';
 const INSERT_REPORT_SUMMARY = 'INSERT INTO reports_summary(test_id, revision_id, report_id, job_id, test_type, phase, start_time, test_name, test_description, test_configuration, notes, last_updated_at) values(?,?,?,?,?,?,?,?,?,?,?,?) IF NOT EXISTS';
 const INSERT_LAST_REPORT_SUMMARY = 'INSERT INTO last_reports(start_time_year,start_time_month,test_id, revision_id, report_id, job_id, test_type, phase, start_time, test_name, test_description, test_configuration, notes, last_updated_at) values(?,?,?,?,?,?,?,?,?,?,?,?,?,?) IF NOT EXISTS';
+const UPDATE_REPORT_BENCHMARK = 'UPDATE reports_summary SET score=?, weights_data=? WHERE test_id=? AND report_id=?';
```

Comment:
wondering if we need it
we can calculate the score dynamically
what is important is to save the weights data
and this can be saved on the query that creates the report first time (insertReport)

as it's also important to update the lastReports table to be able shows this data/calculate it

Reply:
it's a bad idea
if we do it dynamically we need call get test benchmark per each row
forget about it

but need to update last reports with the score.